engine/engine/src/test/label/label.script

```diff
@@ -47,6 +47,11 @@ local function test_vector3(name, expected)
     assert(expected.z == vz)
 end
 
+local function test_single_value(name, expected)
+	local val = go.get("#label", name)
+
+	assert(expected == val)
+end
```

Comment:
looks like the tabs are not 4 spaces ?

Reply:
true. This file cointains the mix of both tabs and spaces. So I changed everything to 4 spaces.